package.json

```diff
@@ -82,10 +78,8 @@
     "fs-extra": "^10.0.1",
     "go-ipfs": "0.12.0",
     "i18next": "^21.6.13",
-    "i18next-electron-language-detector": "0.0.10",
     "i18next-fs-backend": "1.1.4",
     "i18next-icu": "^2.0.3",
-    "intl-messageformat": "^9.11.4",
```

Comment:
This one needs to stay, it is a peer dependency of https://github.com/i18next/i18next-icu#introduction